delta/plugins/archive/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/model/ArchiveReference.scala

```diff
@@ -0,0 +1,77 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.archive.model
+
+import cats.Order
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+
+import java.nio.file.Path
+
+/**
+  * Enumeration of archive references.
+  */
+sealed trait ArchiveReference extends Product with Serializable {
```

Comment:
I thought Archives could contain every possible resource out there including views ?

Reply:
that's correct... what creates this confusion?

Reply:
I understood ResourceReference for data resources only

Reply:
I don't know how to name it better to convey that all resource types are included: schemas, resolvers, storages, views etc.

Reply:
Maybe documenting a little more ?

Reply:
yeah, but can I do that in a different PR to avoid a full re-build?

Reply:
No problem